networking_generic_switch/devices/corsa_devices/corsavfc.py

```diff
@@ -451,6 +447,555 @@ def bridge_detach_tunnel(headers,
     return output
 
 
+#
+# GET DATAPATH
+#
+#   200 OK
+#   303 See Other
+def get_datapath(headers,
+                 url_switch):
+    url = url_switch + ep_datapath
+
+    try:
+        r = requests.get(url, headers=headers, verify=False)
```

Comment:
Actually.. why all the `verify=False`? That's a bit odd.

Reply:
Switches have a self-signed cert on them that is used to enable REST server.